mono/eglib/glib.h

```diff
@@ -107,6 +107,56 @@ struct g_cast
 
 #endif
 
+#ifdef __cplusplus
+
+// Provide for bit operations on enums, but not all integer operations.
+// This alleviates a fair number of casts in porting C to C++.
+
+// Forward declare template with no generic implementation.
+template <size_t> struct mono_size_to_int;
```

Comment:
bikeshed: since this is in glib, maybe a `g` prefix or namespace?

Reply:
Oops, yes.